README.md

```diff
@@ -52,7 +52,8 @@ echo "LOCKBOX_MASTER_KEY=0000000000000000000000000000000000000000000000000000000
 
 ### Database / Cache
 
-1. Create a database called `covidliste_development` using your favorite postgresql GUI or CLI.
+1. Create a database called `covidliste_development` using your favorite
+   postgresql GUI or CLI.
 2. Then run the migrations : `bin/rails db:migrate RAILS_ENV=development`
```

Comment:
We could just run `bin/setup` which handles all of this.

Reply:
:+1: